cypress/e2e/listingDetails/openListings.e2e.ts

```diff
@@ -148,5 +217,36 @@ describe("Listing Details for Open Listings", () => {
         APPLY_BUTTON_TEXT.tl
       )
     })
+
+    /*
+     * If any of these machine translation tests are failing, it could be due to:
+     * 1. Google Translate updated the translation
+     * 2. A Salesforce field changed
+     *
+     * If one of those happen, it should be fine to either update the translation string in this file or find another machine translated field
+     * and test that in the same way it's happening here. These machine translation tests are just to ensure the integration is working, not testing
+     * the specific translation itself.
+     */
+
+    it("machine translations works in Filipino", () => {
+      verifyMachineTranslations("tl", testListings.HABITAT_SALE.id, [
+        INFORMATION_SESSION_SALE_TEXT.tl,
+        PARKING_TEXT.tl,
+      ])
+    })
+
+    it("machine translations works in Chinese", () => {
+      verifyMachineTranslations("zh", testListings.HABITAT_SALE.id, [
+        INFORMATION_SESSION_SALE_TEXT.zh,
+        PARKING_TEXT.zh,
+      ])
+    })
+
+    it("machine translations work in Spanish", () => {
+      verifyMachineTranslations("es", testListings.HABITAT_SALE.id, [
+        INFORMATION_SESSION_SALE_TEXT.es,
+        PARKING_TEXT.es,
+      ])
+    })
```

Comment:
looks good! appreciate the documentation.

one ask. can we please move these tests to their own file? this will allow us to utilize parallelization when running the tests and also then the doc could display just once.